jaxlib/gpu_triton.py

```diff
@@ -15,17 +15,33 @@
 from jaxlib import xla_client
 
```

Comment:
This is suboptimal, because in general I'm trying to maintain the property that CUDA and ROCM could be simultaneously imported in the same JAX program. The code below forces you to pick one: if the ROCM code is present, it will override the CUDA code. Could we avoid that?

(I mean, I admit it's unlikely people do this often, but nothing physically stops you from having a machine with both kinds of GPU, I think?)

Under the upcoming PJRT plugin work, it should be fairly easy to install plugins for both GPUs into JAX at the same time, as well.

It's not absolutely mandatory that we do this, but is there a way we could?

Reply:
This comment wasn't addressed.

Reply:
I think you're right @hawkinsp, if xla can allow multiple stream executor co-exist, we should be capable of achieving it. As a side note, I wonder is TPU and NV-GPUs can coexist in a machine?